### DISCUSSION TITLE
make tags_raw option available again

### DISCUSSION THREAD

**Discussion by Laurent-Cosmic** [FLEXIcontent/flexicontent-cck#1209] [Ideas]
Hello
I would like to ask again for this option tags_raw to allow us to use tags' ID instead of tag name

let me give you an example, I note again yesterday

On my site, there is a tag "bâton" with an accent and a tag "baton" w/o (just 'cos of the language).  When importing the articles, both are mixed-up.
I have 42 articles with this tag: 21 in French and 21 in English - all of them are shown under both tag when my import shown well "bâton" and "baton"

On the other hand, I would like to have a tag "baton" different for some categories, the idea is to have a tag for participants different from the one for the galleries...  
I was obliged to add a Ξ to make it the shorter as possible

By having the import based on ID, it will so easy to make the difference when importing and do not have such issue.

Because, yes a bâton in French is a baton in English, and it's the only word used by professionals so by me too....

Thanks to take a moment to reconsider this request :D 
have a nice day

**Comment by micker** [comment]
hello yes it can be cool but we are concentred to kill all bug before next release no features can be added sorry

**Reply by Laurent-Cosmic** [reply]
of course :D step by step
